### PR TITLE
fix e2e test for assign / unassign

### PR DESCRIPTION
**Pull request by jamil314**
## Description

Link to the GitHub issue: https://github.com/opencrvs/opencrvs-core/issues/9053

The ticket disables some action if not assigned. e2e needs to reflect that

## Checklist
- [ ] I have tested the changes locally, and written appropriate tests
- [ ] I have tested beyond the happy path (e.g. edge cases, failure paths)
- [ ] I have updated the changelog with this change (if applicable)
- [ ] I have updated the GitHub issue status accordingly

### CODE REVIEW

**Comment by adskyiproger** [comment]
Your environment is deployed to https://ocrvs-9053.opencrvs.dev.

File: src/data-seeding/roles/roles.ts [hunk 1]
```diff
@@ -116,6 +116,7 @@ export const roles: Role[] = [
       SCOPES.RECORD_DECLARATION_ARCHIVE,
       SCOPES.RECORD_DECLARATION_REINSTATE,
       SCOPES.RECORD_REGISTRATION_REQUEST_CORRECTION,
+      SCOPES.RECORD_UNASSIGN_OTHERS,
```
Comment (jamil314) [inline src/data-seeding/roles/roles.ts:119]: Will roll it back after [this](https://github.com/opencrvs/opencrvs-core/issues/9102) is done

Comment (cibelius) [reply src/data-seeding/roles/roles.ts:119]: Can you add a comment to the code and reminder on the github issue, so that we remember to remove this? 😊

**Review by cibelius — APPROVED** [review]

**Comment by jamil314** [comment]
Merging because [this](https://github.com/opencrvs/opencrvs-core/actions/runs/14510169615/job/40706950829?pr=9207) passed
